src/parser/mod.rs

```diff
@@ -88,29 +88,56 @@ fn test_parse_value_data_2() -> Result<(), Error> {
     Ok(())
 }
 
-/// Parse a scss file.
+/// Parse a scss file by path.
 ///
 /// Returns a vec of the top level items of the file (or an error message).
-pub fn parse_scss_file(file: &Path) -> Result<Vec<Item>, Error> {
-    do_parse_scss_file(file, &SourceName::root(file.display()))
+pub fn parse_scss_file(path: &Path) -> Result<Vec<Item>, Error> {
```

Comment:
How do you feel about renaming these to `parse_scss_path` and so on, and using these names for the new ones that take `impl Read`? Or do you have a better suggestion for the new names?

I suppose backwards compatibility is more of a concern here.

Reply:
Hm.  Yes, your renaming suggestions makes sense.  But I'm afraid the change is a little less "discoverable" here, as in it may be harder for someone with a compiler error after upgrading to figure out exactly what to do.  But maybe that can be fixed by just adding cross-referencing see-also paragraphs to the doc strings?